src/pages/SplashScreen/SplashScreen.tsx

```diff
@@ -0,0 +1,37 @@
+import { Center } from '@chakra-ui/layout'
+import { Circle, Spinner } from '@chakra-ui/react'
+import { isFirefox } from 'react-device-detect'
+import Orbs from 'assets/orbs.svg'
+import OrbsStatic from 'assets/orbs-static.png'
+import { FoxIcon } from 'components/Icons/FoxIcon'
+import { Page } from 'components/Layout/Page'
+import { colors } from 'theme/colors'
+
+export const SplashScreen = () => {
```

Comment:
There is already a splash screen. Why aren't we just using the one that already exists?